.github/workflows/assignments_test_docker.yml

```diff
@@ -0,0 +1,30 @@
+name: assignments_test_docker
+on: push
+jobs:
+  run_assignments_docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+
+      - name: Configure Makefile.inc
+        working-directory: ./weblab-docker
+        run: |
+          sed -i 's/gtar/tar/' Makefile.inc
+
+      - name: Build Docker image
+        working-directory: ./weblab-docker
+        run: |
+          make
+
+      - name: Execute python script
```

Comment:
The name here can explain what this script does instead of "execute script"

Can we also make this script not to run all the assignments? IIRC, this would take around 20 minutes. Can we just randomly select one or two per folder?